### PR TITLE
add missing methods

### PR DESCRIPTION
**Pull request by BolaNasr**
Add missing methods 
change README
fix some bugs

### CODE REVIEW

File: JumpscaleLib/clients/gitea/GiteaClient.py [hunk 1]
```diff
@@ -1,30 +1,54 @@
...
+from .GiteaOrgs import GiteaOrgs
+from .GiteaMarkdowns import GiteaMarkdowns
+from .GiteaUsers import GiteaUsers
 from JumpscaleLib.clients.gitea.client import Client
 
 
 TEMPLATE = """
 url = ""
 gitea_token_ = ""
+admins = ""
```
Comment (rkhamis) [inline JumpscaleLib/clients/gitea/GiteaClient.py]: why do we have this?

File: JumpscaleLib/clients/gitea/GiteaOrg.py [hunk 2]
```diff
@@ -1,109 +1,130 @@
...
-    {'color': '#bfe5bf', 'name': 'state_inprogress'},
-    {'color': '#bfe5bf', 'name': 'state_question'},
-    {'color': '#bfe5bf', 'name': 'state_verification'},
-    {'color': '#fef2c0', 'name': 'type_bug'},
-    {'color': '#fef2c0', 'name': 'type_task'},
-    {'color': '#fef2c0', 'name': 'type_story'},
-    {'color': '#fef2c0', 'name': 'type_feature'},
-    {'color': '#fef2c0', 'name': 'type_question'}
-]
+
```
Comment (rkhamis) [inline JumpscaleLib/clients/gitea/GiteaOrg.py]: you've removed some functionality here

File: JumpscaleLib/clients/gitea/GiteaClient.py (hunk 1 above)
Comment (BolaNasr) [reply JumpscaleLib/clients/gitea/GiteaClient.py]: i deleted it but i forget to push it ..

File: JumpscaleLib/clients/gitea/GiteaOrg.py (hunk 2 above)
Comment (BolaNasr) [reply JumpscaleLib/clients/gitea/GiteaOrg.py]: i added and tested it :)

**Pull request merged**